lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalProductFunctionFactory.cxx

```diff
@@ -86,6 +87,13 @@ EnumerateFunction OrthogonalProductFunctionFactory::getEnumerateFunction() const
   return tensorizedFunctionFactory_.getEnumerateFunction();
 }
 
+/* Return the collection of univariate orthogonal polynomial families */
+OrthogonalProductFunctionFactory::FunctionFamilyCollection OrthogonalProductFunctionFactory::getFunctionFamilyCollection() const
+{
+  return coll_;
```

Comment:
I tried ; OrthogonalProductFunctionFactory::getFunctionFamilyCollection returns a Collection<OrthogonalUniVariateFunctionFamily> andTensorizedUniVariateFunctionFactory::  has a Collection<UniVariateFunctionFamily> (and no accessor)